pages/blogs/[...slug].vue

```diff
@@ -1,6 +1,18 @@
 <template>
     <div class="container">
-        <BlogsList v-if="slug === '/blogs/'" :blogs="page"/>
+        <BlogsList v-if="slug === '/blogs/'" :blogs="page" :external-news="externalNews"/>
+
+        <div v-else-if="slug === '/blogs/community'">
+            <div class="mt-5 row">
+                <DocsBreadcrumb :slug="slug" :page-list="[]"></DocsBreadcrumb>
```

Comment:
add the blog page in the page-list so you can go back to blog list by clicking on it